.github/workflows/testing.yml

```diff
@@ -9,6 +9,7 @@ on:
     tags:
       - "v*"
   pull_request:
+  workflow_dispatch:
```

Comment:
I just learned about `workflow_dispatch` in #1273.  It lets us trigger a workflow manually.